frontend/src/pages/landing/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import styled from '@emotion/styled';
+import { Link } from 'react-router-dom';
+
+function Footer() {
+  return (
+    <StyledContainer>
+      <StyledTextWrapper>
+        <StyledText>상호명: 핏토링</StyledText>
+        <StyledText>대표자: 주용은</StyledText>
+        <StyledText>이메일: fittoring@gmail.com</StyledText>
+        <StyledText>Ⓒ 2025. fittoring Inc. All right reserved.</StyledText>
+      </StyledTextWrapper>
+      <StyledLink to="https://docs.google.com/forms/d/e/1FAIpQLSfQlaSrxUmU-CKnK6jnp8qLTdGMmLYbff2CZSUmKE09OHN11w/viewform">
+        서비스 문의하기
+      </StyledLink>
```

Comment:
p4: 서비스 문의하기는 구글 폼이라 서비스의 흐름에서 벗어나는 새로운 기능인 것 같아!! 새로운 탭으로 열리게 하는 건 어떻게 생각해? 🧐

```suggestion
      <StyledLink to="https://docs.google.com/forms/d/e/1FAIpQLSfQlaSrxUmU-CKnK6jnp8qLTdGMmLYbff2CZSUmKE09OHN11w/viewform"
      target="_blank">
        서비스 문의하기
      </StyledLink>
```

Reply:
우리 서비스가 아니라서 새로운 탭이 열리는 게 더 적절한거 같아!! 굿굿